CHANGELOG.md

```diff
@@ -76,7 +76,7 @@
 - More price-server scafolding
 - Return pined stream from subscribe
 - Clippy
-- Commit Cargo.lock as we are buiding a binary
+- Commit Cargo.lock as we are building a binary
```

Comment:
You can remove this - rebase against main where I have removed `CHANGELOG.md` from the spell checking.